### DISCUSSION TITLE
Increase max size of notifications dialog

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5856] [Feature Requests]
Current sizing behavior: 

https://github.com/specify/specify7/assets/135047322/ee21e194-7087-48f9-a009-9f9118b7f45a

Would like to be able to at least make it much larger vertically. Horizontal constraints are ok

**Comment by maxpatiiuk** [comment]
why do you need to make it any larger vertically?
what is the use case?

the reason the width is limited is because this way notification message is wrapping over several lines, rather than being in one long line - it's much more readable this way, and looks better

**Comment by grantfitzsimmons** [comment]
It would be nice to have it larger vertically so that you can see more than 2-3 notifications at a single time. It is seemingly arbitrarily restricted to its current height.

**Comment by maxpatiiuk** [comment]
ups. I confused the horizontal and vertical directions 🤦
things make more sense now
I need to sleep more

for developers:
check if disabling height restriction for all dialogs of this kind works here, and doesn't introduce issues in other places where this dialogClassName is used
otherwise, you can overwrite the style for this dialog along (i.e `${dialogClassName....} some-other-style`)
